documentation/guide-how-to-create-a-devon4quarkus-app.asciidoc

```diff
@@ -0,0 +1,224 @@
+:toc: macro
+toc::[]
+
+= How to create a devon4quarkus application
+
+This guide should work as an introduction to the functionality given by https://quarkus.io/[Quarkus] and used in devon4quarkus and show how to build a first application.
+The guide will cover:
+
+* How to build a simple "Hello World" Quarkus app serving a hello endpoint.
+* A short introduction to testing and the development mode in a quarkus app.
+* Packaging the project and running as a self-contained jar file and producing a native executable. 
+
+=== Prerequisites
+To follow the guide and complete it, you need:
+
+* an IDE or an editor to edit files
+* JDK 11+ installed with JAVA_HOME configured appropriately
+* Apache Maven 3.8.1+
+
+
+This guide is heavily inspired by https://quarkus.io/guides/[Quarkus Guides] and the Katacoda tutorials by https://www.katacoda.com/openshift/courses/developing-with-quarkus[OpenShift for Quarkus Developement]. I would recommend also using the Katacoda tutorials to learn about quarkus.
+
+https://quarkus.io/guides/getting-started
+
+https://www.katacoda.com/openshift/courses/developing-with-quarkus/getting-started
+
+
+
+Let's get started.
+
+=== Creating the application
+
+There are multiple ways to start a project. You can browse to https://code.quarkus.io and select some extensions, the build tool for your project, and if you want some starter code. This will provide a project skeleton with the configured project dependencies and also contributes the information to compile the application natively.
+
+To load a new Quarkus application from command line with the Quarkus Maven plugin run the following command:
+
+.Operating system
+. Linux and MacOs +
+`mvn io.quarkus:quarkus-maven-plugin:2.0.2.Final:create \
+    -DprojectGroupId=org.acme \
+    -DprojectArtifactId=getting-started \
+    -DclassName="org.acme.getting.started.GreetingResource" \
+    -Dpath="/hello"`
+
+. Windows +
+    `mvn io.quarkus:quarkus-maven-plugin:2.0.2.Final:create "-DprojectGroupId=org.acme" "-DprojectArtifactId=getting-started" "-DclassName=org.acme.getting.started.GreetingResource" "-Dpath=/hello"`
+ 
+
+it generates the following in the getting-started folder
+
+* the Maven structure
+* an org.acme.getting.started.GreetingResource resource exposed on /hello
+* an associated unit test
+* a landing page that is accessible on http://localhost:8080 after starting the application
+* example Dockerfile files for both native and JVM modes in src/main/docker
+* the application configuration file
+
+Once generarted change your directory with `cd getting-started` and take a look at the *pom.xml*. You will find the import of the Quarkus BOM, allowing you to omit the version on the different Quarkus dependencies. In addition, you can see the *quarkus-maven-plugin* responsible for packaging of the application and also providing the development mode.
+
+[source, xml]
+----
+  <properties>
+<artifact-id>quarkus-universe-bom</quarkus.platform.artifact-id>
```

Comment:
It must be `quarkus.platform.artifact-id` in the opening xml tag. And make sure the indentation is correct